jep/9999/README.adoc

```diff
@@ -0,0 +1,134 @@
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+= JEP-9999: Criteria for selecting "Suggested Plugins"
+
+:toc:
+
+.Metadata
+[cols="2"]
+|===
+| JEP
+| 9999
+
+| Title
+| Criteria for selecting "Suggested Plugins"
+
+| Author
+| link:https://github.com/daniel-beck[Daniel Beck], link:https://github.com/rtyler[R Tyler Croy]
+
+| Status
+| :speech_balloon: Draft
+
+| Type
+| Informational
+
+| Created
+| 2017-09-12
+|===
+
+
+[NOTE]
+====
+This proposal/design predates the Jenkins Enhancement Proposal process and has
+therefore been added as an **Information** type of JEP. Normally this would
+have been submitted as a Draft for as a **Standard** type proposal.
+====
+
+== Abstract
+
+Jenkins 2.0 will include a setup dialog that pops up when Jenkins starts for
+the first time. It intends to help people new to Jenkins to get started
+quickly, and present them with a reasonably sized set of plugins to choose from
+for customizing their instance, of they so choose. To this end, not every
+plugin will be presented here. While the extensibility of Jenkins is one of its
+strengths, exposing new users to the 1200+ available plugins before they've
+even started using Jenkins isn't a reasonable approach.
+
+== Specification
+
+Plugins qualify for inclusion in the list based on a number of criteria:
+
+=== Selection Criteria
+
+* Plugin popularity (both in absolute numbers and growth rate), which can serve as a stand-in for plugin usefulness unless we believe this to be inaccurate for specific reasons (e.g. CVS Plugin and Monitor External Job being installed a lot, but only because they are bundled).
+* Does the plugin integrate well with Jenkins? Does it work e.g. with concurrent builds, in secured Jenkins, and distributed Jenkins (master + agent setup)? People new to Jenkins should not be surprised by a plugin offered here undermining or not working with these features.
+* Is the plugin more than just superficial themeing/UI? While those have their place and are popular, advertising plugins changing the UI out of the box would be very weird (Simple Theme Plugin, Green Balls)
```

Comment:
Green Balls actually includes colorblindness support.

Reply:
Which is only necessary because it introduces the red/green color blindness problem. I don't remember complaints about blue/yellow or other issues in the default setup (perhaps because of brightness differences).